NewSpring/ApollosProject
# [DEV-3880] Refactor Add Prayer Card

## DESCRIPTION

### What does this PR do, or why is it needed?

Part of the `Rosary` design refactor.
- Matches the provided designs for the add prayer card.
- Removes hidden section and animation to the add prayer card.
- Removes `AddPrayerCardConnected` which is no longer necessary with the redesign.
- Adds Button linking to `My Prayers` which has yet to be developed.

### How do I test this PR?

Fire up a sim and make sure it looks right. To see the full effect, you might have to add a view inside the `SafeAreaView` inside the `PrayerMenu` and give it a color. (Which is in the mocks)

Also make sure all the tests pass!

---

> I am affirming this is my _best_ work ([Ecclesiastes 9:10](https://www.bible.com/bible/97/ECC.9.10.MSG))

## TODO


## REVIEW


**Manual QA**


---

> The purpose of PR Review is to _improve the quality of the software._
